internal/policy/root_test.go

```diff
@@ -29,6 +29,66 @@ func TestInitializeRootMetadata(t *testing.T) {
 	assert.Equal(t, []string{key.KeyID}, rootMetadata.Roles[RootRoleName].KeyIDs)
 }
 
+func TestAddRootKey(t *testing.T) {
+	keyBytes, err := os.ReadFile(filepath.Join("test-data", "root.pub"))
```

Comment:
https://github.com/gittuf/gittuf/blob/main/internal/policy/helpers_test.go#L23-L36

feel free to use these with tuf.LoadKey... directly.

Reply:
I have switched it to use the embedded variables